package.json

```diff
@@ -124,6 +124,7 @@
     "date-fns": "2.30.0",
     "docopt": "0.6.2",
     "emoji-toolkit": "9.0.1",
+    "linkifyjs": "4.2.0",
```

Comment:
This is a temp solution while they release the proper fix, right?

Reply:
yeah basically, they just need to update linkifyjs in extension-link and release it, then we can remove this.